### PR TITLE
Please add a new framework for benchmarks - Ditsmod.

### PR DESCRIPTION
**Pull request by KostyaTretyak**
Please add a new framework for benchmarks - [Ditsmod](https://ditsmod.github.io/en/)

### CODE REVIEW

File: javascript/ditsmod/config.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+framework:
+  github: ditsmod/ditsmod
+  website: https://ditsmod.github.io/en/
```
Comment (waghanza) [inline javascript/ditsmod/config.yaml:3]:
```suggestion
  website: ditsmod.github.io/en
```

File: javascript/ditsmod/config.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  version: 2.49.1
```
Comment (waghanza) [inline javascript/ditsmod/config.yaml:4]:
```suggestion
  version: 2.49
```

File: javascript/ditsmod/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "imports": {
+    "#src/*": "./dist/*"
+  },
+  "keywords": [],
+  "author": "Your Name Here",
+  "license": "MIT",
+  "dependencies": {
+    "@ditsmod/body-parser": "^2.12.1",
+    "@ditsmod/core": "^2.49.1",
+    "@ditsmod/router": "^2.7.1"
```
[the remark this answers is not in the document]
Comment (KostyaTretyak) [reply javascript/ditsmod/package.json:20]: No, with `"*"` - it is not a fact that other versions of `body-parser` and `router` will be compatible.

File: javascript/ditsmod/src/main.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import cluster from 'node:cluster';
+import { availableParallelism } from 'node:os';
+import { Application } from '@ditsmod/core';
+
+import { AppModule } from './app/app.module.js';
+
+const numCpus = availableParallelism();
+
+if (numCpus > 1 && cluster.isPrimary) {
```
[the remark this answers is not in the document]
Comment (KostyaTretyak) [reply javascript/ditsmod/src/main.ts:9]: I just didn't expect you to use pm2 in such simple applications.

[the remark this answers is not in the document]
Comment (KostyaTretyak) [reply javascript/ditsmod/src/main.ts:9]: By the way, are the tests run on a multi-core processor?

**Review by waghanza — APPROVED** [review]

**Pull request merged**